test/mitmproxy/contentviews/test_image_parser.py

```diff
@@ -0,0 +1,88 @@
+import io
+
+from mitmproxy.contentviews.image import image_parser
+from mitmproxy.test import tutils
+
+class TestPngParser:
+    def test_png_size(self):
```

Comment:
So after this, all the png tests can be done under a single function? No need of class?